var/spack/repos/builtin/packages/py-gosam/package.py

```diff
@@ -11,17 +11,20 @@ class PyGosam(PythonPackage):
        one-loop amplitudes for multi-particle processes in renormalizable
        quantum field theories."""
 
-    homepage = "https://gosam.hepforge.org"
-    url      = "https://gosam.hepforge.org/downloads/?f=gosam-2.0.4-6d9f1cba.tar.gz"
+    homepage = "https://github.com/gudrunhe/gosam"
+    git      = "https://github.com/gudrunhe/gosam.git"
 
     tags = ['hep']
 
-    version('2.0.4', sha256='faf621c70f66d9dffc16ac5cce66258067f39f686d722a4867eeb759fcde4f44')
+    version('2.1.1', tag='2.1.1')
```

Comment:
Tags always need to have commit IDs to be safe. Can you add it? And is there a reason not to use the packaged binary? (You should be able to add a `url = 'https://github.com/gudrunhe/gosam/archive/refs/tags/2.1.1.tar.gz'` to the package recipe to enable this.)

Reply:
Commit ID - sure, will add. The packaged source you've linked is not usable - one needs to use `https://github.com/gudrunhe/gosam/releases/download/2.1.1/gosam-2.1.1-4b98559.tar.gz`, and this breaks spack's url guessing.